clients/ansible/test/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
This is a really new version of CMake, do we actually need this? If not, we should set the value to something easy to obtain, like 2.8.